modules/swagger-codegen/src/main/resources/php-symfony/Controller.mustache

```diff
@@ -135,6 +135,12 @@ class Controller
         if (null === $exception) {
             return null;
         }
+        
+        if ($this->container->get('kernel')->getEnvironment() === 'prod') {
```

Comment:
Do not test against this, check for debug mode.

Reply:
Personally, I also prefer the debug flag.

Reply:
It makes sense seeing this is debug information. Production environment can also be run in debug mode.

Reply:
Okay i will change this in using debug

Reply:
Done changes in 
60a2c7aebf188434ff4713cb4bdce593a1870b8b